src/rp2_common/hardware_clocks/clocks.c

```diff
@@ -163,11 +163,15 @@ bool clock_configure(enum clock_index clk_index, uint32_t src, uint32_t auxsrc,
 }
 /// \end::clock_configure[]
 
-void clocks_init() {
+void clocks_init(void) {
     clocks_reinit(0, 0);
 }
 
 void clocks_reinit(uint32_t sleep_en0, uint32_t sleep_en1) {
+	// Determine whether SYS_CLK is already running
+    bool init_sys_clock =  	((sleep_en0 & CLOCKS_SLEEP_EN0_CLK_SYS_DEPENDANTS_BITS) == 0) &&
```

Comment:
Can you replace spaces with tabs please, and fix up the indentation? :slightly_smiling_face: